multicluster/controllers/multicluster/clusterset_controller.go

```diff
@@ -97,13 +98,17 @@ func (r *ClusterSetReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	// if update,
 	//    if leader - nothing to do in inbound mode
 	//    if member - if leaders have changed handle accordingly, else nothing to do.
-	if !r.IsLeader {
+
+	// skip creating remote cluster manager in leader only cluster for inbound mode.
+	// otherwise, we need initilize both LocalClusterManager and RemoteClusterManager
+	if r.IsLeader && !r.IsMember {
+		r.createLocalClusterManager(clusterSet)
+	} else {
 		err = r.updateMultiClusterSetOnMemberCluster(clusterSet)
 		if err != nil {
 			return ctrl.Result{}, err
 		}
-	} else {
-		r.createOrUpdateMultiClusterSetOnLeaderCluster(clusterSet)
+		r.createLocalClusterManager(clusterSet)
```

Comment:
please help to move `r.createLocalClusterManager(clusterSet)` above before `err = r.updateMultiClusterSetOnMemberCluster(clusterSet)`

Reply:
Done here https://github.com/antrea-io/antrea/pull/3041/commits/55262e110b252c20b3cd8b9fb9052f3c7af2608f#diff-e689d782ce89eea671205881811ddd5aa958531d9f67a60f58002929b408802fR113